fdbcli/TenantCommands.actor.cpp

```diff
@@ -996,9 +1012,9 @@ std::vector<const char*> tenantHintGenerator(std::vector<StringRef> const& token
 	if (tokens.size() == 1) {
 		return { "<create|delete|deleteId|list|get|getId|configure|rename>", "[ARGS]" };
 	} else if (tokencmp(tokens[1], "create") && tokens.size() < 5) {
-		static std::vector<const char*> opts = { "<NAME>",
-			                                     "[tenant_group=<TENANT_GROUP>]",
-			                                     "[assigned_cluster=<CLUSTER_NAME>]" };
+		static std::vector<const char*> opts = {
+			"<NAME>", "[tenant_group=<TENANT_GROUP>]", "[assigned_cluster=<CLUSTER_NAME>]", "[ignore_capacity_limit]"
```

Comment:
We should also update `tenantGenerator` above

Reply:
Done